packages/fxa-content-server/server/lib/configuration.js

```diff
@@ -594,6 +594,44 @@ const conf = module.exports = convict({
     env: 'STATIC_RESOURCE_URL',
     format: 'url'
   },
+  subscriptions: {
+    allowedLanguages: {
+      default: ['en-US'],
+      doc: 'Which languages are allowed to access subscription features?',
+      env: 'SUBSCRIPTIONS_ALLOWED_LANGUAGES',
+      format: Array,
+    },
+    enabled: {
+      default: false,
+      doc: 'Indicates whether subscriptions APIs are enabled',
+      env: 'SUBSCRIPTIONS_ENABLED',
+      format: Boolean,
+    },
+    managementClientId: {
+      default: 'YOU MUST CHANGE ME',
```

Comment:
Let's use a default of `59cceb6f8c32317c`. I'll open a PR with an entry on the oauth server.

Reply:
I used `98e6508e88680e1a` here just to get it working, pending a resolution for #881 to introduce the real OAuth client